.github/workflows/android.yml

```diff
@@ -165,4 +165,4 @@ jobs:
       - name: Code push release
         run: |
           echo "Code push release target vervion `git rev-list origin/3493-code-push HEAD --first-parent --count`"
-          npx appcenter codepush release-react --token "${{ env.APPCENTER_TOKEN }}" -a "${{ env.APPCENTER_NAME }}" -d Staging  -t `git rev-list origin/3493-code-push HEAD --first-parent --count`
+          npx appcenter codepush release-react --token "${{ env.APPCENTER_TOKEN }}" -a "${{ env.APPCENTER_NAME }}" -d Production  -t `git rev-list origin/3493-code-push HEAD --first-parent --count`
```

Comment:
shouldn't 'code push' be instead of build & deploy to the appcenter ?

Reply:
Yes, I commented out the App center deployment action

Reply:
@alt-a1r  Build Android Release and Build Universal APK **are still uncommented**
why ? **hot code push should build the app** instead. and this should happen faster than 20-25min for the full build

Reply:
this is the main purpose of the hot code push. if it doesn't works that way - we don't need it at all

Reply:
@alt-a1r plese read the article https://blog.logrocket.com/implementing-codepush-react-native/